src/net_processing.cpp

```diff
@@ -1037,14 +1053,12 @@ unsigned int LimitOrphanTxSize(unsigned int nMaxOrphansSize)
         nNextSweep = nMinExpTime + ORPHAN_TX_EXPIRE_INTERVAL;
         if (nErased > 0) LogPrint(BCLog::MEMPOOL, "Erased %d orphan tx due to expiration\n", nErased);
     }
+    FastRandomContext rng;
```

Comment:
14626: this is from 14624 which was backported partially via #3923. Probably makes sense to backport leftovers now instead of introducing (unrelated) changes via 14626?